guide/payments/request.md

```diff
@@ -0,0 +1,471 @@
+---
+layout: guide
+title: Requesting bitcoin
+description: An introduction to how requesting bitcoin works.
+nav_order: 3
+parent: Payments
+permalink: /guide/payments/request/
+redirect_from:
+ - /guide/onboarding/receive/
+main_classes: -no-top-padding
+# image: /assets/images/guide/payments/request/header.svg
+---
+
+<!--
+
+Illustration source
+
+- https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=337%3A2159
+
+-->
+
+# Requesting bitcoin
+
+To request bitcoin, the receiver must provide the sender with a payment request. Several payment request formats are available, both with Bitcoin and Lightning, each having its unique use cases and trade-offs. Payment requests can be single-use or reusable. This page covers the design considerations around generating, presenting, and sharing payment requests as well as receiving bitcoin.
+
+## Requesting entry points
+
+When a user needs to request bitcoin, there are several ways in which this can be initiated including:
+
+- Generating single-use payment requests
+- Using reusable payment requests
+- Importing a withdraw request
+
+Below is an overview of these various entry points:
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/Home.png"
+   retina = "/assets/images/guide/payments/request/Home@2x.png"
+   layout = "shadow"
+   caption = "Home screens can provide quick access to entering request flows."
+   alt-text = "Screen showing home screen number pad entry."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
```

Comment:
Considering Lighting also has addresses (via "Lightning addresses"), I don't think we should use the terms "Invoice" and "Address" to differentiate between Lightning and on-chain in the tabs in these screens.

Also don't think it's useful to show the invoice code ("lnbc1..."), pretty much gibberish to 99.99% of people.